swarm.cabal

```diff
@@ -254,7 +254,7 @@ library
                       astar                         >= 0.3 && < 0.3.1,
                       blaze-html                    >= 0.9.1 && < 0.9.2,
                       boolexpr                      >= 0.2 && < 0.3,
-                      brick                         >= 1.10 && < 1.11,
+                      brick                         >= 2.1.1 && < 3.0,
```

Comment:
Let's make this `< 2.2` instead of `< 3.0`, according to the PVP there's no guarantee we will be compatible with other 2.x releases.

The same comment applies to the upper bounds on `vty`, `vty-crossplatform`, and `vty-windows` / `vty-unix` below.

Reply:
Done!